Formula/libsixel.rb

```diff
@@ -15,14 +17,13 @@ class Libsixel < Formula
 
   depends_on "meson" => :build
   depends_on "ninja" => :build
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
+  depends_on "libpng"
```

Comment:
These aren't actually linked in existing/new bottles due to some meson build script issues, but they do get linked in HEAD. Could optionally make HEAD-only for now.

HEAD build fails on PNG with fallback STB so just adding `libpng` as dependency.